docs/make.jl

```diff
@@ -0,0 +1,20 @@
+using Documenter, Dualization
+
+makedocs(
+    modules = [Dualization],
+    doctest  = false,
+    clean    = true,
+    format   = Documenter.HTML(),
```

Comment:
I would recommend using these options: https://github.com/JuliaOpt/MathOptInterface.jl/blob/master/docs/make.jl#L5-L8

Reply:
You have twice the field format now

Reply:
hehehe ça arrive